std/bitmanip.d

```diff
@@ -3536,6 +3537,7 @@ unittest
         T     = The integral type to convert the first $(D T.sizeof) bytes to.
         endianness = The endianness to write the bytes in.
         range = The range to append to.
+        value = The value to append.
```

Comment:
DDoc for a symbol `foo` highlights `foo` in the documentation description, assuming that every use of the word `foo` is a reference to the symbol itself. The same is done for parameter names. The underscore tells DDoc that the word is used in its original meaning, not as a reference to a symbol.